modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbTableStorage.java

```diff
@@ -586,7 +602,7 @@ private List<String> getExistingCfNames() {
 
             // even if the database is new (no existing Column Families), we return the names of mandatory column families, that
             // will be created automatically.
-            return existingNames.isEmpty() ? List.of(META_CF_NAME, PARTITION_CF_NAME, HASH_INDEX_CF_NAME) : existingNames;
+            return existingNames.isEmpty() ? List.of(META_CF_NAME, PARTITION_CF_NAME, GC_QUEUE_CF_NAME, HASH_INDEX_CF_NAME) : existingNames;
```

Comment:
Just thoughts: It would be more convenient to iterate with an enum.

Reply:
Not really, we don't have `SORTED_INDEX` and `UNKNOWN` here

Reply:
I see, thanks!